packages/webapp/src/utils/redirect-view-handler.js

```diff
@@ -0,0 +1,8 @@
+const addRedirectViewUsed = (handler) => {
```

Comment:
this is preferred to having to add the boolean to all routes that return an error following a form submission - i.e. it would be easy to miss.